src/rascal/representations/calculator_spherical_invariants.hh

```diff
@@ -550,6 +547,10 @@ namespace rascal {
                 continue;
               }
 
+              auto && soap_neigh_gradient_by_species_pair{
+                  soap_neigh_gradient[spair_type]};
+              // soap_neigh_gradient_by_species_pair.setZero();
+
               // TODO(max) is there a symmetry here we can exploit?
```

Comment:
ping @max-veit, is this still relevant?

Reply:
Yup. I think the gradients code can be simplified quite a bit, but I'm still not sure how.